game/health_checks.py

```
@@ -0,0 +1,48 @@
+    ChessPuzzle,
+    Achievement,
+    LessonProgress,
+    OpeningProgress,
+)
+
+
+def check_database():
+    try:
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT 1")
+        return True
```

Ruff (0.15.17)

[warning] Consider moving this statement to an `else` block

(TRY300)